meetings/2023-09-06.md

```diff
@@ -0,0 +1,164 @@
+* TBL: But it's hairy.
+* RW: In Europe your code does not need to be ??? I would not spare those 3 lines that we have to write to save hassle for people. 
+* SC: Pre-proposal: replacing MIT license with W3C software without notice. We can come back to making the text clear in the specs themselves. 
+* TBL: Good to have the intention in the charter. 
+* SC: Let's make that change request in the PR. Agreement is to merge the charter PR. 
+* RW: Main issue remaining is, if people contribute and commit to the CLA it requires notification so at some point we can have a web form where people give up the notification for those who originally contributed text to the spec. When is it recharterting? Current charter, when does it apply? IS it in effect?
+* SC: It was effective September 1st. We're talking about the CG charter. First charter. Amendments can come if we want to make changes.
+* RW: How can I get evidence of the commitment? The intended result is clear. Probability that somebody comes and says they did not commit is close to zero, but to do it correctly we must record commitment. 
+* WT: Does this differ from the process where we get a FSA? 
+* RW: We can do that on the same step. 
+* SC: That's on the CG page somewhere. There's a way to tell who made a commitment and who didn't. 
+* SC: Objections to making the change to W3C license without notice and following up with specific text to introduce in text for next releases? 
+* SC: We'll merge the charter and if we find an error we can have another discussion and amend. 
+* WT: +1
+* TBL: +1
+* eP: +1
+
+DECISION: apply suggested change, SC to merge. 
+
+* eP: Rigo mentioned something about test suite. Does the charter already address or do we need to do something special?
+* RW: Test suite is double licenced, you don't need to do anything. If you make claims of conformance you cannot alter. 
+* eP: For test suite it is better not to use MIT but the one recommended by W3C. 
+* RW: Not very different. For the test suite, for claims of conformance or intend to use it for claims of conformance, you cannot take altered code, because then your claims would be false. Copyright is an additional kind of helper, not main target. 
+* WT: The sentences in the charter about this is exactly what W3C has about double licensing, so we don't deviate from that. 
+* SC: RW, in the license section we have to "note that". Do you recommend that we keep that text in or remove? 
+* RW: First of all you say WG instead of CG which is wrong. I think per W3C process this is a community process so normal process doesn't apply. I don't thin kit makes a legal different but would look nicer if you note that artifacts that become part of this group's TR will fall under W3C software license.
+* WT: The notes are meant to indicate that independent of the license we use, these items will be under a WG's license. 
```

Comment:
```suggestion
* RW: First of all, you say WG instead of CG, which is wrong. I think, per W3C process, this is a community process, so normal process doesn't apply. I don't think it makes a legal different, but would look nicer if you note that artifacts that become part of this group's TR will fall under W3C software license.
* WT: The notes are meant to indicate that, independent of the license we use, these items will be under a WG's license. 
```